lib/global-styles/experimental-default-global-styles.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Moved from the top-level. So posting here the old contents for comparison:

```
{
	"color": {
		"primary": "#52accc",
		"background": "white",
		"text": "black"
	}
}
```

Reply:
Note that I've removed `primary` here as the color hooks at https://github.com/WordPress/gutenberg/pull/21021 don't implement it.